lms/djangoapps/course_blocks/transformers/start_date.py

```diff
@@ -71,7 +71,7 @@ def collect(cls, block_structure):
 
     def transform_block_filters(self, usage_info, block_structure):
         # Users with staff access bypass the Start Date check.
-        if usage_info.has_staff_access:
```

Comment:
@ormsbee, @nasthagiri: Is it better to have a conditional like this, or to remove the transformer from the transformer stack for some uses? I'm not really sure how we wanted to handle conditional transforms like this.